packages/reapit-connect-management/src/components/ui/apps/app-installation-manager.tsx

```diff
@@ -106,72 +105,101 @@ const AppInstallationManager: React.FC<AppInstallationManagerProps> = ({ app }:
   }
 
   return (
-    <Section>
-      <FlexContainerBasic className={styles.flexContainerSpaceBettwen}>
-        <H5>Installation</H5>
-        <Button
-          variant="primary"
-          disabled={!submitButtonEnabled}
-          loading={false}
-          onClick={() => setShowConfirmModal(true)}
-        >
-          Submit
-        </Button>
-      </FlexContainerBasic>
-      <p className="mb-4">
-        <i>Please select the type of installation you require for this app:</i>
-      </p>
-      <div className="field field-checkbox mb-0 control">
-        <input
-          className="checkbox"
-          type="radio"
-          id={`${app.id}-${WHOLE_ORG}`}
-          checked={appInstallationType === WHOLE_ORG}
-          onChange={handleOnCheckboxChange(setAppInstallationType, WHOLE_ORG)}
-        />
-        <label className="label" htmlFor={`${app.id}-${WHOLE_ORG}`}>
-          Install for the whole of your organisation
-        </label>
-        <div className="form-subheading mb-4">
-          <i>This will grant the app access to all data for all users and offices across your organisation</i>
+    <>
+      <Section>
```

Comment:
I added this section for uninstalling - basically it should just render the number of offices you are installed for if you have installations / org installations. If you don't tells you to go to the next section.

I thought this made most sense because I _think_ you can just re-use the same modal, actions and logic 

_I think_ 😬